website/docs/r/automation_software_update_configuration.html.markdown

```diff
@@ -49,160 +49,160 @@ resource "azurerm_automation_software_update_configuration" "example" {
+- `advanced_week_days` - (Optional) List of days of the week that the job should execute on. Only valid when frequency is `Week`.
 
-* `advanced_month_days` - (Optional) List of days of the month that the job should execute on. Must be between `1` and `31`. `-1` for last day of the month. Only valid when frequency is `Month`.
+- `advanced_month_days` - (Optional) List of days of the month that the job should execute on. Must be between `1` and `31`. `-1` for last day of the month. Only valid when frequency is `Month`.
 
-* `monthly_occurrence` - (Optional) List of `monthly_occurrence` blocks as defined below to specifies occurrences of days within a month. Only valid when frequency is `Month`. The `monthly_occurrence` block supports fields as defined below.
+- `monthly_occurrence` - (Optional) List of `monthly_occurrence` blocks as defined below to specifies occurrences of days within a month. Only valid when frequency is `Month`. The `monthly_occurrence` block supports fields as defined below.
 
 ---
 
 The `monthly_occurrence` block supports the following:
 
-* `day` - (Required) Day of the occurrence. Must be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
+- `day` - (Required) Day of the occurrence. Must be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
 
-* `occurrence` - (Required) Occurrence of the week within the month. Must be between `1` and `5`. `-1` for last week within the month.
+- `occurrence` - (Required) Occurrence of the week within the month. Must be between `1` and `5`. `-1` for last week within the month.
 
 ## Attributes Reference
 
 In addition to the Arguments listed above - the following Attributes are exported:
 
-* `id` - The ID of the Automation Software Update Configuration.
+- `id` - The ID of the Automation Software Update Configuration.
 
-* `error_code` - The Error code when failed.
+- `error_code` - The Error code when failed.
 
-* `error_meesage` - The Error message indicating why the operation failed.
+- `error_meesage` - The Error message indicating why the operation failed.
 
 ## Timeouts
 
 The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
 
-* `create` - (Defaults to 30 minutes) Used when creating the Automation.
-* `read` - (Defaults to 5 minutes) Used when retrieving the Automation.
-* `update` - (Defaults to 10 minutes) Used when updating the Automation.
-* `delete` - (Defaults to 10 minutes) Used when deleting the Automation.
+- `create` - (Defaults to 30 minutes) Used when creating the Automation.
+- `read` - (Defaults to 5 minutes) Used when retrieving the Automation.
+- `update` - (Defaults to 10 minutes) Used when updating the Automation.
+- `delete` - (Defaults to 10 minutes) Used when deleting the Automation.
```

Comment:
These don't need to/shouldn't be replaced with dashes, could you set these back?

Reply:
> These don't need to/shouldn't be replaced with dashes, could you set these back?
Its weird, I used vs code to make change, and once I change from dash to star, and save it it auto save as dash...
Anyway I just changed these back via online editor. Please review.